chirp/drivers/ga510.py

```diff
@@ -285,6 +278,30 @@ def get_features(self):
                           (400000000, 480000000)]
         return rf
 
+    def validate_memory(self, mem):
+        msgs = chirp_common.CloneModeRadio.validate_memory(self, mem)
+
+        # _mem = self._memobj.memory[mem.number]
+        _msg_duplex = 'Duplex must be "off" for this frequency'
+        _msg_offset = 'Only simplex or +5MHz offset allowed on GMRS'
+
+        if self.MODEL == "RA85":
+            if mem.freq not in bandplan_na.ALL_GMRS_FREQS:
+                if mem.duplex != "off":
+                    msgs.append(chirp_common.ValidationWarning(_msg_duplex))
+            if mem.freq in bandplan_na.GMRS_HIRPT:
+                if mem.duplex and mem.offset != 5000000:
+                    msgs.append(chirp_common.ValidationWarning(_msg_offset))
+                if mem.duplex and mem.duplex != "+":
+                    msgs.append(chirp_common.ValidationWarning(_msg_offset))
+        if self.MODEL == "RA685":
+            if not ((mem.freq >= self.vhftx[0] and mem.freq < self.vhftx[1]) or
+                    (mem.freq >= self.uhftx[0] and mem.freq < self.uhftx[1])):
+                if mem.duplex != "off":
+                    msgs.append(chirp_common.ValidationWarning(_msg_duplex))
+
+        return msgs
+
```

Comment:
I guess I wasn't really paying close enough attention to the others, but we really should put these two chunks (without the if) on the subclasses. Meaning only override validate_memory for those special subclasses instead of doing the `if self.MODEL`. The latter works, but it's much less conventional in python, even though we have a lot (too much) of it in chirp. Not a huge deal, and I already let the others in so I won't hold this up, but ... my OCD...